network/dns_frame.ksy

```diff
@@ -0,0 +1,121 @@
+meta:
+  id: dns_packet
+  title: DNS (No support for Auth-Name + Add-Name for simplicity)
+  application: https://www.ietf.org/rfc/rfc1035.txt
```

Comment:
We were discussing in https://github.com/kaitai-io/kaitai_struct/issues/59 to use `doc-ref` for that (or just use a comment right now). "Application" is meant to identify a particular piece of software that this format is used for. Given that network packages are highly standartized and used by tens of thousands (if not hundreds) of different software / firmware / hardware implementations everywhere, probably it's better to keep clean of "application".